client/app/bundles/course/group/pages/GroupShow/GroupTableCard.jsx

```diff
@@ -35,11 +43,31 @@ const translations = defineMessages({
     defaultMessage:
       'This group has no members! Manage groups to assign members now!',
   },
+  manageOneGroup: {
+    id: 'course.group.GroupShow.CategoryCard.manageOne',
```

Comment:
Be careful with the key. After you paste it, dont forget to change the id.. See the patterns. We cant have duplicate ids across our codebase for intl.